### ISSUE TITLE
convert the model from pytorch to onnx to caffe2, but get a lower accuracy than before

### ISSUE THREAD

**Issue by Joeywzr** [pytorch/pytorch#20154]
When I convert the model from pytorch into caffe2, the top1 and top5 accuracy rate on ILSVRC12 dataset became smaller.

The procedure for converting is:

1.convert pytorch into onnx IR
2.convert onnx IR into caffe2

I have tested many pretrained models. All of the accuracy decreased a lot.

![EDFhQJ.png](https://s2.ax1x.com/2019/05/06/EDFhQJ.png)

I don't know what caused the problem. May the internal differences between the pytorch and caffe2 make ?

**Comment by ezyang** [comment]
@fmassa suggests that the problem might be related to preprocessing differences between the PyTorch and Caffe2. It's hard to say without seeing the code however.

**Comment by dtmoodie** [comment]
I have found similar results when exporting to onnx to be consumed by tensorrt.
I ensure that the same preprocessing occurs before passing the data into pytorch and into tensorrt.
